gpxconv

```diff
@@ -679,7 +679,12 @@ for(my $i=0; $i <= $#TIM; $i++) {
                                         ($theta_diff >= 0 ? "more than  " : "less than -").MAX_SENSIBLE_ANGLE_DIFF."°)");
     }
     elsif($Spd > MAX_SENSIBLE_SPEED) {
-      ignoring_trkpt("speed is more than ".MAX_SENSIBLE_SPEED." km/h");
+      ignoring_trkpt(
+        sprintf(
+            'speed %.2f is more than MAX_SENSIBLE_SPEED km/h',
```

Comment:
NIce. Yet please use `SPD_PRECISION` instead of `%.2f`,
replace also `MAX_SENSIBLE_SPEED` by `SPD_PRECISION`,
and re-phrase the message a little, such that the line becomes
```
'speed = '.SPD_PRECISION.' km/h is more than '.SPD_PRECISION,
```